index.html

```diff
@@ -263,12 +277,14 @@ <h2 class="about-p-text" >About Me</h2>
                             see and have a project you need coded, 
                             don’t hesitate to contact me.
                         </p>
-                        <div class="btn-holder">
+                        <div class="btn-holder resume-button">
                             <button class="btn">Get My Resume</button>
                         </div>
                         
-                        <img class="about-me-banner" src="./assets/images/About Me Background.svg" alt="Aboutmebanner">
-                        
+                    </article>
+                    <img class="about-me-banner" src="./assets/images/About Me Background.svg" alt="Aboutmebanner">
+                    <img class="about-me-desktop" src="./assets/images/aboutme-dekstop23.svg" alt="aboutmedesktop">
```

Comment:
- [ ] Please 🙏🏻  use the image as background because using background images for decoration in web development has advantages over using the "IMG" tag, especially when the images are not the main content. Background images provide flexibility 💪🏻  in scaling ⚖️ and positioning, separate decorative elements from essential content, optimize performance, ensure consistency 💯 , and allow for creative layering effects. They enhance visual appeal without compromising the overall structure and accessibility 👓  of the main content.